src/components/elements/FlexibleGatewayCard.tsx

```diff
@@ -0,0 +1,121 @@
+import { useRouter } from "next/router";
+import { Card, Col, Row, Typography } from "antd";
+import NodeCard from "./FlexibleNodeCard";
+import {
+  getFormattedLastSeen,
+  getFormattedLastSeenDate,
+  getFormattedVoltageData,
+} from "../presentation/uiHelpers";
+import { GATEWAY_MESSAGE, ERROR_MESSAGE } from "../../constants/ui";
+import styles from "../../styles/Home.module.scss";
+import cardStyles from "../../styles/Card.module.scss";
+import { Gateway, SensorTypeCurrentReading, GatewaySensorTypeNames, NodeSensorTypeNames } from "../../services/AppModel";
+
+interface GatewayProps {
+  gateway: Gateway;
+  index: number;
+}
+
+function findCurrentReadingWithName(gateway: Gateway, name: SensorTypeCurrentReading["sensorType"]["name"]) : SensorTypeCurrentReading | undefined{
+  return gateway.currentReadings?.find((sensorTypeReading) => name===sensorTypeReading.sensorType.name);
+}
+
+const GatewayCardComponent = (props: GatewayProps) => {
+  const { index, gateway } = props;
+  const { Text } = Typography;
+
+
+  const router = useRouter();
+  // todo - use urlBuilder to create the correct URL
+  const gatewayUrl = `/${gateway.id.gatewayDeviceUID}/details`;
+  const handleCardClick = (e: React.MouseEvent<HTMLElement>) => {
+    e.preventDefault();
+    // eslint-disable-next-line @typescript-eslint/no-floating-promises
+    router.push(gatewayUrl);
+  };
+
+  
+  // we hard code some of the gateway readings for now. We will make this more open-ended in future with the site redesign
+  const formattedGatewayVoltage = getFormattedVoltageData(
+    findCurrentReadingWithName(gateway, GatewaySensorTypeNames.VOLTAGE)?.reading?.numericValue || undefined
```

Comment:
The build fails on this line with this error.

> Type error: Property 'numericValue' does not exist on type 'Reading'.

Reply:
Fixed...it was a last minute change that I hadn't done completely.